.github/workflows/create_release.yml

```diff
@@ -29,8 +29,8 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ~/.m2/repository
-          key: ${{ runner.os }}-jdk-11-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: ${{ runner.os }}-jdk-11-maven-
+          key: ${{ runner.os }}-jdk-17-maven-${{ hashFiles('**/pom.xml') }}
```

Comment:
was a change @reta made in the other projects, which luckily wasn't fatal in the last release, but should be fixed anyway ;)